### PR TITLE
Coverity issues fixes

### PR DESCRIPTION
**Pull request by oschwaldp-oci**

### CODE REVIEW

File: dds/DCPS/MultiTopicDataReaderBase.cpp [hunk 1]
```diff
@@ -284,14 +284,14 @@ void MultiTopicDataReaderBase::data_available(DDS::DataReader_ptr reader)
...
+          ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) MultiTopicDataReaderBase::data_available:")
+            ACE_TEXT(" failed to obtain DataReaderImpl.")));
         }
       }
     }
   } catch (const std::runtime_error& e) {
     if (OpenDDS::DCPS::DCPS_debug_level) {
-      ACE_ERROR((LM_ERROR, "(%P|%t) MultiTopicDataReaderBase::data_available: "
+      ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) MultiTopicDataReaderBase::data_available: ")
         "%C", e.what()));
```
Comment (mitza-oci) [inline dds/DCPS/MultiTopicDataReaderBase.cpp]: The wchar problem is still present on this line.  I suggest making the %C part of line 294 instead of 295.

Comment (oschwaldp-oci) [inline dds/DCPS/MultiTopicDataReaderBase.cpp]: thanks. fixed.

File: dds/FACE/FaceTSS.cpp [hunk 2]
```diff
@@ -69,18 +69,26 @@ using OpenDDS::FaceTSS::Entities;
...
+  try {
+    int status = parser.parse(configuration_file);
+    if (status != 0) {
+      ACE_ERROR((LM_ERROR,
+        ACE_TEXT("(%P|%t) ERROR: Initialize() ")
+        ACE_TEXT("Parser::parse () returned %d\n"),
+        status));
+      return_code = INVALID_PARAM;
+    }
+    else {
```
Comment (mitza-oci) [inline dds/FACE/FaceTSS.cpp]: Check formatting here (and a few other similar ones in this file).

File: dds/DCPS/WaitSet.cpp [hunk 3]
```diff
@@ -72,13 +72,17 @@ ReturnCode_t WaitSet::detach_conditions(const ConditionSeq& conds)
 {
   ACE_GUARD_RETURN(ACE_Recursive_Thread_Mutex, g, lock_,
                    RETCODE_OUT_OF_RESOURCES);
+  try {
```
Comment (mitza-oci) [inline dds/DCPS/WaitSet.cpp]: Not sure I agree with this one.  I'd rather not change code just to make Coverity happy -- there's no way to get an exception here.

**Pull request merged**